.github/workflows/config-template.yaml.j2

```diff
@@ -77,7 +81,14 @@ hub:
 
       c.JupyterHub.authenticator_class = MarketplaceOAuthenticator
       c.JupyterHub.admin_access = True
+      c.JupyterHub.extra_handlers = [
+        (
+          r'/heartbeat',
+          HeartbeatHandler,
+        ),
+      ]
```

Comment:
This is a good demonstrator for an additional handler, but I don't think we need since the `/hub/health` endpoint already exists.